scripts/start.sh

```diff
@@ -4,4 +4,8 @@ set -e
 export GUNICORN_CONF=${GUNICORN_CONF:-/gunicorn_conf.py}
 
 # Start Gunicorn
-exec gunicorn -b 0.0.0.0:${UVICORN_PORT} -k uvicorn.workers.UvicornWorker --workers=${UVICORN_NUM_WORKERS} -c ${GUNICORN_CONF} ${UVICORN_ASGI_APP}
\ No newline at end of file
+if [[ -z "${ENABLE_MONITORING}" && "${ENABLE_MONITORING}" = "true" ]]; then
+  exec ddtrace-run gunicorn -b 0.0.0.0:${UVICORN_PORT} -k uvicorn.workers.UvicornWorker --workers=${UVICORN_NUM_WORKERS} -c ${GUNICORN_CONF} ${UVICORN_ASGI_APP}
+else
+  exec gunicorn -b 0.0.0.0:${UVICORN_PORT} -k uvicorn.workers.UvicornWorker --workers=${UVICORN_NUM_WORKERS} -c ${GUNICORN_CONF} ${UVICORN_ASGI_APP}
+fi
```

Comment:
Change the env var name to `OPAL_ENABLE_MONITORING`
Please notice that in OPAL, confi will prefix env var names with `OPAL_`